src/handlebars/views/report.hbs

```diff
@@ -6,15 +6,80 @@
 
```

Comment:
You can get rid of the form-wrap div since the submit button is inside the form after all :)